webview/src/plots/components/comparisonTable/cell/ComparisonTableBoundingBoxColorFilter.tsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react'
 
+export const getColorFilterId = (imgAlt: string, color: string) =>
+  `${imgAlt.replace(/[\s()]/g, '-')}-${color.slice(1)}`
```

Comment:
Ids need to be unique, so we can't just rely on the color text. I created an id based off the imgAlt, removing anything that was against the HTML5 spec.

Reply:
There is a file `webview/src/util/ids.ts`. Might be a good id to move this there and to make it generic (`getIdFromTextAndColor`).